PureBasicIDE/ProcedureBrowser.pb

```diff
@@ -150,7 +150,12 @@ Procedure UpdateProcedureList()
     
     ; Lock the list update, as on GTK2 it's really slow
     ; Note: we may not call SetGadgetState() after this!
-    CompilerIf #CompileLinux
+    
+    ;- Author : mk-soft
+    ;  Date   : 22.05.2020
+    ;  Bugfix : Linux Crash with g_object. Disabale optimize
+    
+    CompilerIf 0; #CompileLinux
```

Comment:
If the code within the disabled `CompilerIf` block no longer makes sense, the complete block should be removed.